### PR TITLE
Added flush

### PR DESCRIPTION
**Pull request by rstz**

### CODE REVIEW

File: nativeio_async_wrapper.js [hunk 1]
```diff
@@ -58,6 +58,11 @@ mergeInto(LibraryManager.library, {
       return NativeIOWrapper.fileDescriptorToFileHandle[fd].getLength();
     },
 
+    // Flushes changes on the fd onto disk.
```
Comment (fivedots) [inline nativeio_async_wrapper.js]: Nit.: I would say "to the disk"

File: wrapped.cpp [hunk 2]
```diff
@@ -45,6 +46,9 @@ int main() {
   double length  = NativeIO_GetLength(f);
   EM_ASM({console.log("Got new length of file:", $0)}, length);
 
+  NativeIO_Flush(f);
+  EM_ASM({console.log("Flushed file descriptor"});
```
Comment (fivedots) [inline wrapped.cpp]: Nit.: I would just say flushed file

**Review by fivedots — APPROVED** [review]

File: wrapped.cpp (hunk 2 above)
Comment (rstz) [reply wrapped.cpp]: Done

File: nativeio_async_wrapper.js (hunk 1 above)
Comment (rstz) [reply nativeio_async_wrapper.js]: Done

**Pull request merged**